solutions/stubs/src/clientCallback/handlers/clientCallback.ts

```diff
@@ -147,6 +147,7 @@ const getTokenRequestBody = async ({
     iss: client.client_id,
     aud: process.env["API_TOKEN_ENDPOINT_URL"],
     jti: crypto.randomUUID(),
+    sub: "urn:fdc:gov.uk:default",
```

Comment:
The sub is sent in the client_assertion, which is then returned in the access_token and sent to the journeyoutcome API (where it is a [required field](https://github.com/govuk-one-login/account-components/blob/main/solutions/api/src/lambda/journeyoutcome/utils/validateJourneyOutcomeJwtClaims.ts#L25)) 

I'm not sure what the best way to get the user id is